rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md

```diff
@@ -44,15 +41,19 @@ If you believe the issues being raised are not important, please check the secti
 ### Modify Visual Studio Analysis Ruleset
 
 The goal is to develop a shared ruleset across projects. (Currently this is just the default settings). This will ensure the same standard and quality of code is maintained across all of the company's projects.
-Any project specific rules should be documented in "\_Instructions-CodeHealth.docx" which is to be kept in the solution.
-Please also copy the current version number of this rule into the "\_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
+Any project specific rules should be documented in "_Instructions-CodeHealth.docx" which is to be kept in the solution.
+Please also copy the current version number of this rule into the "_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
```

Comment:
We typically store docs in Markdown files or project wiki. Not word documents

Check use cases isnt already covered by https://www.ssw.com.au/rules/awesome-documentation/#good-example--the-8-important-documents

Reply:
Yeah this seems like a super old and frankly poo recommendation. I'd just replace it with a link to the rule Brady linked

Reply:
Done - changed this to follow the files from the [https://www.ssw.com.au/rules/awesome-documentation/](https://www.ssw.com.au/rules/awesome-documentation/#good-example--the-8-important-documents), and mentioned the rule itself